.github/workflows/windows-test.yml

```
@@ -0,0 +1,30 @@
+        uses: actions/setup-go@v5.5.0
+        with:
+          go-version: '>=1.24.2'
+          
+      - name: Build and Run Grype
+        run: |
+          gci env:* | sort-object name
+          dir C:\Users\RUNNER~1\AppData\Local\Temp\grype*.*
+          go version
+          go run .\cmd\grype db update
+          dir C:\Users\RUNNER~1\AppData\Local\Temp\grype*.*
+                
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)